code/modules/mob/living/simple_animal/hostile/clown.dm

```diff
@@ -310,3 +310,27 @@
 	melee_damage_upper = 20
 	armour_penetration = 30
 	environment_smash = ENVIRONMENT_SMASH_WALLS//no escape
+
+/mob/living/simple_animal/hostile/clown/stacked
```

Comment:
i opened this at school and now i am frightened. I want my mom to pick me up early.

Reply:
join in the fun!join in the fun!join in the fun!

Reply:
stop, i am filled with fear